app/Ptr/PtrValidateRdns.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Packages\Rdns\App\Ptr;
+
+use Packages\Rdns\App\Ptr\DnsRecordService;
+
+class PtrValidateRdns {
+
+  /**
+   * @var DnsRecordService
+   */
+  protected $dns;
+
+  /**
+   * @param DnsRecordService $dns
+   */
+  public function __construct(DnsRecordService $dns){
+    $this->dns = $dns;
+  }
+    /**
+     * @return bool
+     */
+    public function validate($input, $domain) {
```

Comment:
the tabbing seems off in this function compared to the other ones. Please make sure your editor is setup to use 2 spaces for tabbing and auto-format using prettier on every save. There are VS Code extensions for this. We should document that setup process in VS Code onboarding docs if we don't have it already documented, which I don't think we do.

Also please make sure each of the arguments have types and the function has a return type. Also `$input` is not a clear domain name.